src/argilla_server/search_engine/commons.py

```diff
@@ -188,6 +192,18 @@ def es_mapping_for_question_suggestion(question: Question) -> dict:
     }
 
 
+def es_script_for_delete_user_response(user: User) -> str:
+    return f'ctx._source["responses"].remove("{es_path_for_user(user)}")'
+
+
+def es_path_for_user(user: User) -> str:
```

Comment:
We're already on the search engine scope so we don't need to add `es` prefix to everything (in the case that you want to change them, not sure if it's easy or worth it):

```suggestion
def path_for_user(user: User) -> str:
```